### DISCUSSION TITLE
I'm on Xubuntu 22.04

### DISCUSSION THREAD

**Discussion by bonelifer** [randoragon/rsid3#16] [Q&A] [answered]
I'm on Xubuntu 22.04, how do I install this?

**Comment by randoragon** [comment, marked as answer]
Hi, you can install it either from [crates.io](https://crates.io/crates/rsid3) or this GitHub.

I recommend GitHub, because you are guaranteed the latest version then. Crates.io only gets updated on new release, which does not happen quite as often.

### GitHub

```sh
git clone https://github.com/randoragon/rsid3
cd rsid3
cargo install --path .
```

### crates.io

```sh
cargo install 'rsid3@1.0.0-alpha.1'
```
It is necessary to pin the specific version at the moment, because there is no stable release yet.

---

In both methods, the binary will be placed in `$CARGO_HOME/bin`. It should then be available to you, assuming you have `$CARGO_HOME/bin` in your `PATH`.
